### ISSUE TITLE
Tooltip still works even after calling dispose method.

### ISSUE THREAD

**Issue by honux** [FezVrasta/popper.js#169]
### CodePen demo

http://codepen.io/anon/pen/egLXaQ

### Steps to reproduce the problem

1.  Open the site code pen example
2.  Hover the mouse on the ref element sometimes.

### What is the expected behavior?

The tooltip should be disabled after calling the "dispose" method. 

### What went wrong?

The tooltip still works, even after calling the dispose method.

**Comment by FezVrasta** [comment]
Oh well.. at least it's the first time someone complains that something actually works 😂

I will look into it as soon I have a minute.

**Comment by FezVrasta** [comment]
So, looks like `dispose` can work only if the tooltip has been shown at least once... Because `dispose` looks for `this.tooltipNode`, which is available only after the tooltip has been shown once.

I haven't time to fix it at the moment because I focus my spare time on the core, any contributions are more than welcome.

**Comment by aristidesfl** [comment]
Has this been fixed or just closed?

**Comment by FezVrasta** [comment]
Actually I thought it had been fixed.

**Comment by aristidesfl** [comment]
Would bump priority of this since it sounds like basic functionality.

**Comment by FezVrasta** [comment]
Sorry I give low priority to Tooltip.js because it's just a side-project to showcase Popper.js.

Feel free to adopt the issue and send a PR if you need this fixed asap.

**Comment by FezVrasta** [comment]
This has been fixed

**Comment by waltermaldonado** [comment]
Could you please take a look in it again? I am having the exact same issue with Tooltip.js 1.3.1.
